namelist.wps

```diff
@@ -28,11 +30,11 @@
  !!!!!!!!!!!!!!!!!!!!!!!!!!!! IMPORTANT NOTE !!!!!!!!!!!!!!!!!!!!!!!!!!!!
  !
  geog_data_res = 'default','default',
- dx = 30000,
- dy = 30000,
+ dx = 15000,
+ dy = 15000,
  map_proj = 'lambert',
- ref_lat   =  34.83,
- ref_lon   = -81.03,
+ ref_lat   =  33.00,
+ ref_lon   = -79.00,
  truelat1  =  30.0,
  truelat2  =  60.0,
  stand_lon = -98.0,
```

Comment:
@kkeene44 Should we change stand_lon to be the same as ref_lon? Can you add the new domain plot?

Reply:
@weiwangncar Not sure how I missed this message from you. Yes, I agree. I made the change and committed it.